config/secrets.yml.example

```diff
@@ -50,3 +71,13 @@ production:
       signature_public_key: |
         ssm(sso_signature_public_key)
       encryption_private_key: ssm(sso_encryption_private_key)
+  content:
+      abl_url: <%= ENV['OPENSTAX_CONTENT_ABL_URL'] || 'https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-book-list.json' %>
```

Comment:
the other prod credentials are either using that `{{ }}` notation, which i think is handled [here](https://github.com/openstax/unified-deployment/blob/cb2cbddce1f99de134b2acd643e45c161d6de7ae/apps/highlights/scripts/deployment.rb#L122), or they're loading from ssm, in either case, i think these content parameters need to be updated to work correctly

for each of these params we're going to need to figure out how to populate it... and it might be easier if there were less of them... for instance i don't actually think you need all the parameters about the direct s3 access

Reply:
S3 stuff is only needed as a fallback to grab the latest version from S3 if the REX version lookup fails, which seems pretty unlikely, so I could probably remove it.

I totally overlooked the substitution and ssm stuff, that script link helps clarify things for me... seems like I need to poke around the deployment repo.